package.json

```diff
@@ -38,9 +38,10 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
+    "@types/extend": "^3.0.1",
```

Comment:
I don't think we should add this dependency it is not required - I think the problem is basically that we need to run the latest generator to get source code that doesn't require this type (see e.g. https://github.com/IBM/platform-services-node-sdk/pull/40).

I think instead we could merge the `2.4.5` upgrade commit alongside the latest generator source (i.e. close this PR and add the core change commit onto that branch when it is ready).

Reply:
Sounds good, I'll close this PR.